app/controllers/cars_controller.rb

```diff
@@ -38,12 +38,25 @@ def destroy
     @car.destroy!
   end
 
+  # PUT /cars/:id/update_status
+  def update_status
+    # binding.pry
+    new_status = params[:status].to_i
+    unless [0, 1].include?(new_status)
```

Comment:
missing line break before and after block

Reply:
can you add a model level validation so that the status value can be only either 0 or 1